src/rhsmlib/dbus/service_wrapper.py

```diff
@@ -18,6 +18,11 @@
 import logging
 
 from rhsmlib.dbus import server
+from subscription_manager import logutil
+
+# FIXME: Should this be logging to the same file from a different process?
```

Comment:
[No, it shouldn't.](http://stackoverflow.com/questions/15096090/python-separate-processes-logging-to-same-file)  I think we're going to need to use an rhsmlib.log file.